Integrations/integration-Cisco-umbrella.yml

```diff
@@ -567,7 +570,9 @@ script:
     outputs:
     - contextPath: Domain.Name
       description: Domain name
-    - contextPath: Domain.Score
+    - contextPath: Domain.Related.Name
+      description: Related domain name
+    - contextPath: Domain.Related.Score
```

Comment:
If you have malicious you need to add important - please add it after @aviadl fixes to important are finished and merged